### ISSUE TITLE
1.7.10 waila conflict

### ISSUE THREAD

**Issue by Rikonah** [sinkillerj/ProjectE#1126]
not sure if it is waila, project e, or waila harvestability, but these three seem to be incompatable.  not saying that they really need to work together, I personally chose project e over waila, but thought I would let you know so you can post a possibly incompatable mods list.

**Comment by williewillus** [comment]
This is entirely too vague. Are you crashing? What's the issue? I need more information.

**Comment by sinkillerj** [comment]
And for that matter logs, please follow the reporting guidelines.

**Comment by koalalord9999** [comment]
(Using all 3 in my 1.7.10 custom self-pack nothing wigs out, works fine for me)

**Comment by Rikonah** [comment]
Sorry, had a limited time to report last time.  What I noticed, and not sure if it is just the version of waila I was using, when I would go to use the transmutation table it would close my single player map and say that the server crashed.  when I removed waila it suddenly would allow me into the transmutation table again.  I almost think it is a graphical error, something conflicting for me, but I don't code so I don't know.  As I said before though, as much as waila is helpful, I have missed the functionality that Project E brings in.  If I had a crash log to give I would to help.  I don't think this is a priority or even anything to worry about because it could be a very limited amount of people affected by this, and it is very minute.

I just wanted you to be aware in case someone starts saying that their game keeps saying server closed on a single player map, this could be the cause.  Thank you for taking the time to view this, and sorry about this rather, lengthy..post.
